### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by Snikem**
1-ая домашка

### CODE REVIEW

File: src/main/java/edu/hw1/Task5.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    public static int toDescendant(int num) {
+        int n = num;
+        String result = "";
+        int i = 0;
+        while (n != 0) {
+            result = (n % 10 + (n % 100 / 10)) + result;
+            n /= 100;
+            i++;
```
Comment (vasilmik) [inline src/main/java/edu/hw1/Task5.java:29]: зачем этот i ?

File: src/test/java/edu/hw1/TestTask6.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package edu.hw1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class TestTask6 {
+    @Test
+    @DisplayName("Постоянная Капрекара")
+    void testCountK() {
```
Comment (vasilmik) [inline src/test/java/edu/hw1/TestTask6.java:10]: Проверьте corner case

File: src/main/java/edu/hw1/Task8.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                if (arr[i][j] == 0) {
+                    continue;
+                }
+                for (int g = 0; g < 8; g += 2) {
+                    try {
+                        if (arr[i + shifts[g]][j + shifts[g + 1]] == 1) {
+                            //System.out.println(i + " " + j);
+                            return false;
+                        }
+                    } catch (ArrayIndexOutOfBoundsException exception) {
```
Comment (vasilmik) [inline src/main/java/edu/hw1/Task8.java:27]: В этой задаче не может быть exception

Comment (Snikem) [reply src/main/java/edu/hw1/Task8.java:27]: Добрый день, я могу задать вам вопросы здесь?
